File: backend/Models/Vehicle.cs
namespace Backend.Models;
public class Vehicle
{
    public int Id { get; set; }
    public int AccountId { get; set; }
    public string Make { get; set; }
    public string Model { get; set; }
    public int Year { get; set; }
    public string LicensePlate { get; set; }
    public string? Color { get; set; }
    public string? Type { get; set; }
    public string? Status { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }
}

File: frontend/src/interfaces/vehicle.ts
export interface Vehicle {
  id: number
  accountId: number
  make: string
  model: string
  year: number
  licensePlate: string
  color?: string
  type?: string
}

export interface VehicleFormData {
  make: string
  model: string
  year: number
  licensePlate: string
  color?: string
  type?: string
}
